Peer.Domain/Commands/Show.cs

```diff
@@ -25,7 +26,7 @@ public class Show
             _sorter = sorter;
         }
 
-        public async Task<Result<None, ShowError>> ShowAsync(ShowArguments _, CancellationToken token = default)
+        public async Task<Result<None, ShowError>> ShowAsync(ShowArguments args, CancellationToken token = default)
```

Comment:
Seems odd that this would go from `_` to `args` but then have no changes that use `args`.

Reply:
Result of moving things around a bit (should be rmd)